src/js/_enqueues/lib/link.js

```diff
@@ -103,8 +103,10 @@
 
 		open: function( editorId, url, text ) {
 			var ed,
+				$wrap = $( '#wpwrap' ),
 				$body = $( document.body );
 
+			$wrap.attr( 'aria-hidden', 'true' );
```

Comment:
Nitpick: Should the `open` function set a variable to use it only once? The `close` function has `$( '#wpwrap' )` with the `aria-hidden` attribute, though that function does not assign _any_ variables for reuse. The variable might help readability a little.

Reply:
I'm inclined to avoid code churn for legacy code blocks like this; I chose to do it this way for consistency with existing patterns in both functions. Could go either way, but it's not clear to me that it makes any practical difference.

Reply:
Adjusting in commit to avoid creating a single-use variable, per this comment.